requirements.txt

```diff
@@ -52,3 +52,6 @@ django-rest-swagger==0.3.2
 djangorestframework-bulk==0.2.1
 pyjwt==1.4.0
 pyjwe==0.1.6
+
+# Admin requirements
+django-extensions==1.5.9
```

Comment:
If this is an admin requirement, then it should be in `requirements/admin.txt`.

That said, are you sure this is a production requirement? IIRC, the extensions are for development environments only, in which case this shouldn't be installed on production.